src/mlpack/methods/ann/brnn_impl.hpp

```diff
@@ -60,6 +60,44 @@ BRNN<OutputLayerType, MergeLayerType, MergeOutputType,
   /* Nothing to do here. */
 }
 
+template<typename OutputLayerType, typename MergeLayerType,
+         typename MergeOutputType, typename InitializationRuleType,
+         typename... CustomLayers>
+template<typename OptimizerType>
+typename std::enable_if<
+      HasMaxIterations<OptimizerType, size_t&(OptimizerType::*)()>
+      ::value, void>::type
+BRNN<OutputLayerType, MergeLayerType, MergeOutputType,
+    InitializationRuleType, CustomLayers...>::WarnMessageMaxIterations
+(OptimizerType& optimizer, size_t samples) const
+{
+  if (optimizer.MaxIterations() < samples &&
+      optimizer.MaxIterations() != 0)
+  {
+    Log::Warn << "The optimizer's maximum number of iterations "
+              << "is less than the size of the dataset, the "
```

Comment:
```suggestion
              << "is less than the size of the dataset; the "
```

Sorry, this is really pedantic. :)